tests/integration/suite_test.go

```diff
@@ -89,7 +89,7 @@ var _ = BeforeSuite(func() {
 
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
 		Scheme:                 scheme.Scheme,
-		MetricsBindAddress:     ":8080",
+		MetricsBindAddress:     ":8085",
```

Comment:
Using another port for metrics server as i was getting errors with 8080 being in use

Reply:
Revert it. Not related to this PR.